kousa/lib/socket_handler.ex

```diff
@@ -361,8 +362,8 @@ defmodule Kousa.SocketHandler do
     {:ok, state}
   end
 
-  def handler("edit_room_name", %{"name" => name}, state) do
```

Comment:
because `DogeHouse` runs in "prod", you can't rename/delete any handlers (because old clients will still call these). Add a comment about deprecating them then add new handlers

Reply:
so a comment would be sufficient? Won't the old clients still fail?